internal/server/v0/internalplace/golden/get_place_page_data/asm.sample.json

```diff
@@ -11177,182 +11453,505 @@
-            "1899": 5,
-            "1900": 3,
-            "1901": 6,
-            "1902": 3,
-            "1903": 2,
-            "1904": 3,
-            "1905": 2,
-            "1906": 5,
-            "1907": 3,
-            "1908": 5,
-            "1909": 6,
-            "1910": 2,
-            "1911": 4,
-            "1912": 6,
-            "1913": 3,
-            "1914": 1,
-            "1915": 4,
-            "1916": 9,
-            "1917": 1,
-            "1918": 2,
-            "1919": 3,
-            "1920": 3,
-            "1921": 3,
-            "1922": 1,
-            "1923": 4,
-            "1924": 3,
-            "1925": 1,
-            "1926": 3,
-            "1927": 1,
-            "1928": 3,
-            "1929": 2,
-            "1930": 1,
-            "1931": 2,
-            "1932": 5,
-            "1933": 5,
-            "1934": 5,
-            "1935": 2,
-            "1936": 6,
-            "1937": 4,
-            "1938": 3,
-            "1939": 3,
-            "1940": 3,
-            "1941": 4,
-            "1942": 3,
-            "1943": 2,
-            "1944": 4,
-            "1945": 4,
-            "1946": 4,
-            "1947": 6,
-            "1948": 4,
-            "1949": 4,
-            "1950": 4,
-            "1951": 2,
-            "1952": 3,
-            "1953": 7,
-            "1954": 6,
-            "1955": 5,
-            "1956": 3,
-            "1957": 6,
-            "1958": 5,
-            "1959": 8,
-            "1960": 5,
-            "1961": 3,
-            "1962": 1,
-            "1963": 4,
-            "1964": 7,
-            "1965": 4,
-            "1966": 1,
-            "1967": 3,
-            "1968": 7,
-            "1969": 5,
-            "1970": 6,
-            "1971": 12,
-            "1972": 6,
-            "1973": 3,
-            "1974": 3,
-            "1975": 3,
-            "1976": 7,
-            "1977": 5,
-            "1978": 4,
-            "1979": 6,
-            "1980": 5,
-            "1981": 4,
-            "1982": 2,
-            "1983": 4,
-            "1984": 4,
-            "1985": 7,
-            "1986": 3,
-            "1987": 3,
-            "1988": 7,
-            "1989": 5,
-            "1990": 1,
-            "1991": 2,
-            "1992": 3,
-            "1993": 3,
-            "1994": 4,
-            "1995": 5,
-            "1996": 3,
-            "1997": 2,
-            "1998": 7,
-            "1999": 5,
-            "2000": 4,
-            "2001": 3,
-            "2002": 7,
-            "2003": 6,
-            "2004": 7,
-            "2005": 7,
-            "2006": 2,
-            "2007": 5,
-            "2008": 6,
-            "2009": 2,
-            "2010": 4,
-            "2011": 3,
-            "2012": 4,
-            "2013": 1,
-            "2014": 2,
-            "2015": 2,
-            "2016": 7,
-            "2017": 6,
-            "2018": 4,
-            "2019": 4,
-            "2020": 11,
-            "2021": 8
+            "1851-06": 1,
```

Comment:
Not blocking, but can we prefer P1Y for ibtracs?

Reply:
Separately, should we report "0" as an observation for other months without cyclone observations? In general, we have a longstanding problem with how we're displaying these charts, because we can't treat missing dates as 0.

https://autopush.datacommons.org/place/geoId/06?category=Environment#Earthquakes

Reply:
@juliawu -- the P1Y change needs ranking-config updates to [mixer](https://github.com/datacommonsorg/mixer/blob/master/internal/server/ranking/ranking.go#L118), [pipeline](https://source.corp.google.com/piper///depot/google3/datacommons/prophet/flume_generator/ranking_helper.cc;l=128?q=f:datacommons%20f:ranking_helper.cc)

@beets -- that is a very good/important point.  Should we have a tracker issue?  (Might require changes to a bunch of hazard datasets)

Reply:
+1, I've filed a bug @pradh